packages/highcharts-theme/src/highcharts-options-salt.ts

```diff
@@ -0,0 +1,41 @@
+import type { Options } from "highcharts";
+// Properties relating to color will be overrided due to styledMode: true
+
+export const highchartsOptionsSalt: Options = {
+  chart: {
+    styledMode: true,
+  },
+  tooltip: {
+    headerFormat: "<span>{point.key}</span><br/>",
+    pointFormat:
+      '<span>{series.name}: </span><span class="value">{point.y}</span>',
+  },
+  legend: {
+    layout: "vertical",
```

Comment:
Same here, how much are we interested in defining defaults over styling?

Reply:
My point was really, is this actually something we want to set as a default? It feels like a bit of an overreach